perception/map_based_prediction/src/map_based_prediction_node.cpp

```diff
@@ -423,7 +423,8 @@ bool MapBasedPredictionNode::checkCloseLaneletCondition(
   // of the lanelets that are below max_dist and max_delta_yaw
   if (
     lanelet.first < dist_threshold_for_searching_lanelet_ &&
-    abs_norm_delta < delta_yaw_threshold_for_searching_lanelet_) {
+    (abs_norm_delta < delta_yaw_threshold_for_searching_lanelet_ ||
+     abs_norm_delta > M_PI - delta_yaw_threshold_for_searching_lanelet_)) {
```

Comment:
```suggestion
    (delta_yaw_threshold_for_searching_lanelet_< abs_norm_delta  ||
     abs_norm_delta < delta_yaw_threshold_for_searching_lanelet_ )) {
```
This one might be better. nitpick comment 🙏

Reply:
@yukkysaito Is it okay to remove M_PI from the condition?

Reply:
sorry, it's misstake
```suggestion
    (M_PI  - delta_yaw_threshold_for_searching_lanelet_< abs_norm_delta  ||
     abs_norm_delta < delta_yaw_threshold_for_searching_lanelet_)) {
```

Reply:
done